src/Umbraco.Cms.Api.Management/Controllers/Preview/EndPreviewController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Api.Management.Controllers.Preview;
+
```

Comment:
```suggestion

[ApiVersion("1.0")]
```

This is important as otherwise we see those endpoints as v2 as well